input/pagecontent/security-authentication.md

```diff
@@ -8,11 +14,67 @@ The Trust over IP framework makes use of Trusted Sources or Trusted Third Partie
 * Flexible, trust networks can be created whenever needed.
 * Simple in its core, VCs and VPs are manifested as commonly used signed JSON objects (JWS).
 
-### NUTS and Verifiable Credentials.
-The implementation for Trust over IP by SCP is NUTS. The NUTS nodes are able to implement the role of both holder and verifier within the trust construct of SCP. Each service in the SCP has its own NUTS-node that acts both wallet and interface for trust exchange. 
+### Verifiable Credential to FHIR Organization
+Given the organization authenticates itself using a Verifiable Credential, the attributes contained within can be used to create or derive the FHIR Organization resource.
+This identification mechanism is flexible: any type of Verifiable Credential (and even non-Verifiable Credentials) can be used, as long as parties agree on a trusted source and presentation format.
+This section outlines the options that are known to be available.
+
+#### NutsUraCredential
+The `NutsUraCredential` ([JSON-LD context](https://nuts.nl/credentials/2024)) is a Verifiable Credential that contains the following properties:
+- URA number: a unique identifier for healthcare organizations in the Netherlands.
+- Name: the name of the organization.
+- City: locality of the organization.
+
+The credential is to be issued by a party trusted in the particular SCP use case, as currently, there's no governing body that issues it.
+It can be represented in both JWT and JSON-LD format. The JSON-LD version of the credential can look as follows:
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://nuts.nl/credentials/2024"
+  ],
+  "id": "did:web:example.com:issuer#1",
+  "type": ["VerifiableCredential", "NutsUraCredential"],
+  "issuer": "did:web:example.com:issuer#1",
+  "issuanceDate": "2024-01-01T00:00:00Z",
+  "expirationDate": "2025-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:web:example.com:holder#1",
```

Comment:
did:web:example.com:holder#1 naar did:web:examplehospital.nl ?

Reply:
I prefer registered example domains (`example.com`, `example.org` and `example.net`), since it'll prevent sending/using an actual domain (that was used as test data) accidentally in production. We could use a subdomain though? (`hospital.example.com`)